app/controllers/users_controller.rb

```diff
@@ -44,14 +45,36 @@ def destroy
     redirect_to user_url
   end
 
+  def following 
+    @title = "Following"
+    @user = User.find(params[:id])
+    @user = @user.followed_users.paginate(page: params[:pate], per_page: 5)
```

Comment:
1. @user before the = (equal sign) should be plural
2. :page mispelling
```suggestion
    @users = @user.followed_users.paginate(page: params[:page], per_page: 5)
```